### ISSUE TITLE
Cannot use output of `sh` to detect novelty

### ISSUE THREAD

**Issue by bhb** [juxt/mach#29]
I may be misunderstanding something, but I would expect that I could call a command via `sh` and then use the stdout to detect novelty (my use case is using `git` to detect if a submodule is out of date).

However, in my testing, I can't seem to get the value of stdout after running a command.

Version: juxt/mach@1.4.0

I've got a toy example below where I just try to print out the value of an `echo`.

Machfile.edn:

```clojure
{
 example/example1 {novelty (do
                             (prn "result is:" (mach.core/sh ["echo" "hello"]))
                             false
                             )}

 }
```

Expected: The return value of `sh` is the value of standard out from running the "echo" command.
Actual: The return value of `sh` is nil.

**Comment by jonpither** [comment]
Thanks for raising - we've been battling the best way to solve this given the async nature of the node spawn process call. We have a fix that is in 1.4.1 that should work for you.

**Comment by bhb** [comment]
@jonpither Thanks for the fast turnaround on this fix!
